pytorch_lightning/trainer/training_loop.py

```diff
@@ -690,7 +694,7 @@ def run_training_batch(self, batch, batch_idx):
             signal=0,
             grad_norm_dic=grad_norm_dic,
             batch_log_metrics=batch_log_metrics,
-            training_step_output_for_epoch_end=opt_closure_result.training_step_output_for_epoch_end
+            training_step_output_for_epoch_end=training_step_output_for_epoch_end_list
```

Comment:
this here is now a list of outputs. This means we need to make sure the method which called this processes that.

Reply:
sounds good :smile: